blocks/suggest/suggest.js

```diff
@@ -73,6 +73,11 @@ provide(BemDom.decl(this.name, {
                 this._isMenuEmpty = null;
                 this._hoveredItem = null;
             }
+        },
+
+        'disabled' : function(modName, modVal) {
+            this._input.setMod(modName, modVal);
+            this._menu.findBlockOn('menu').setMod(modName, modVal);
```

Comment:
`findBlockOn` adds extra DOM operation that will be invoked on every `setMod` / `delMod`. It feels that to store a reference to `menu` instance in the `this` will be a better solution. At least until we have dealt with #5.